src/providers/github.ts

```diff
@@ -14,8 +11,18 @@ export class GitHub implements OAuth2Provider {
 		clientSecret: string,
 		options?: {
 			redirectURI?: string;
+			/** Configuration for usage with [GitHub Enterprise Server](https://docs.github.com/en/enterprise-server/get-started). */
+			enterprise?: {
```

Comment:
Can you rename it to just `enterpriseDomain` to be consistent with the other providers? Or was there a reason to make it an object?

Reply:
not exactly, i will rename it